tasks/Tasks/rhods-groups-config.robot

```diff
@@ -0,0 +1,28 @@
+*** Settings ***
+Documentation       Task to modify groups
+...                 How to change groups dynamically:
+...                 ./run_robot_test.sh --test-case ./tasks/Tasks/rhods-groups-config.robot
+...                 -i Custom-Groups --test-variable ADMIN_GROUPS:<admin_groups>
+...                 --test-variable ALLOWED_GROUPS:<allowed_groups>
+
+
+Library             OpenShiftLibrary
+Resource            ../Resources/Configurations/groups/rhods-groups-config.resource
+
+
+*** Variables ***
+${ADMIN_GROUPS} =       rhods-admins
+${ALLOWED_GROUPS} =     rhods-users
+
+
+*** Test Cases ***
+Configure Custom RHODS Groups Configuration
```

Comment:
This should be a regular test (in a test suite) suite not a task in my opinion

Reply:
could you check PR #208 ? maybe they are testing the same scenario

Reply:
I used Test Cases, but it was by mistake, I wanted to use Tasks instead, because my purpose is to create a task and not a test, sorry for being confusing, I'm changing it. This task is meant to be used to modify groups as part of the cluster configuration after installing ODS, and all of this should be done before executing the tests. Also, this fix was needed to avoid the problems that we were having with new groups, and solving it with this task, allow us to modify the groups without changing the tests, by just modifying the Configmap when configuring the cluster, and therefore, it can be used directly by Jenkins. And under this kind of thinking, we can add more tasks for configurations  and this will give us the power to fully customize our clusters depending on our needs.
@jgarciao, I think this also answers your above comment about the folder that should contain it. 
@bdattoma this might clarify that your tests and the task can perfectly coexist, test the behaviour is also needed.

Reply:
oh okay Pablo, if it is a Task is ok :)

Reply:
Hey @pablofelix , if I understand correctly a Task cannot be used inside a TC like a keyword can, if that's the case I think it'd be more useful to have this as a keyword rather than a task.
I say that because if this were a keyword, we would be able to dinamically change the group configuration before/after/during a specific TC execution.
As an example, I would personally turn this into a keyword and use it as test setup and test teardown for this suite: https://github.com/red-hat-data-services/ods-ci/blob/master/tests/Tests/500__jupyterhub/jupyterhub-user-access.robot

Reply:
a keyword to do that is present in PR #208 if you want (it may seem like I'm advertising my PR ahaha it's not my intent). You'll just need to set the Suite variable with the desired new groups